tests/Microsoft.CodeAnalysis.Testing/Microsoft.CodeAnalysis.CodeFix.Testing.UnitTests/MultipleCodeFixesTests.cs

```diff
@@ -75,6 +75,43 @@ class TestClass {{
             }.RunAsync();
         }
 
+#if !NETCOREAPP1_1 && !NET46
+        [Fact]
+        public async Task TestDefaultSelectionNestedFixers()
+        {
+            var testCode = @"
+class TestClass {
+  int field = [|0|];
+}
+";
+            var fixedCode = $@"
+class TestClass {{
+  int field = 1;
+}}
+";
+
+            // The batch code fix provider does not support nested code actions.
+            // https://github.com/dotnet/roslyn/issues/43044
+            var batchFixedCode = testCode;
```

Comment:
📝 This test could have been written two different ways to work around the bug:

1. Disable the Fix All test
2. Use a different expected outcome for the normal and Fix All tests (specifically, the result is the same as the input which means no change was applied)

I opted for the second approach for one important reason: this approach ensures that when we finally do update the tests to a version of Roslyn that supports the batch fixer in this scenario, the test will start to fail. This test will be correct at the earliest point where it is valid to do so (no earlier or later), even if the person who fixes the bug isn't the same person who eventually updates the dependency in this project.

/cc @Evangelink @bartonjs @stephentoub who may be interested in how I approached a test scenario like this.